src/__tests__/patients/related-persons/RelatedPersons.test.tsx

```diff
@@ -196,4 +196,39 @@ describe('Related Persons Tab', () => {
       expect(history.location.pathname).toEqual('/patients/123001')
     })
   })
+
+  describe('EmptyList', () => {
+    const patient = {
+      id: '123',
+      rev: '123',
+    } as Patient
+
+    const user = {
+      permissions: [Permissions.WritePatients, Permissions.ReadPatients],
+    }
+
+    beforeEach(async () => {
+      jest.spyOn(PatientRepository, 'find')
+      mocked(PatientRepository.find).mockResolvedValue({
+        fullName: 'test test',
+        id: '123001',
+      } as Patient)
+
+      await act(async () => {
+        wrapper = await mount(
+          <Router history={history}>
+            <Provider store={mockStore({ patient, user })}>
+              <RelatedPersonTab patient={patient} />
+            </Provider>
+          </Router>,
+        )
+      })
+      wrapper.update()
+    })
+
+    it('should display a warning if patient has no related persons', () => {
+      const warning = wrapper.find(Alert)
+      expect(warning).toBeDefined()
+    })
+  })
 })
```

Comment:
I've also added a test to ensure the warning message is displayed when a patient has no related persons.